static/app/components/sidebar/newOnboardingStatus.tsx

```diff
@@ -96,7 +97,7 @@ export function NewOnboardingStatus({
   }, [onShowPanel, isActive, walkthrough, markDoneTaskAsComplete, organization]);
 
   useEffect(() => {
-    if (totalRemainingTasks !== 0 || isActive) {
+    if (totalRemainingTasks !== 0 || hasQuickStartBeenCompleted.current) {
```

Comment:
We can trigger the analytics to track whether all groups have been completed before closing the sidebar. This change does that

Reply:
Those analytics will fire on every component mount once all tasks are done, is this intentional?

Reply:
oh I missed that! good catch 🙏 this commit solves the issue https://github.com/getsentry/sentry/pull/79911/commits/0ea80e62c47861c53fc313a9a77329f27d847c72